lib/pgBackRest/Protocol/Helper.pm

```diff
@@ -95,15 +175,7 @@ sub protocolGet
 
```

Comment:
CSHANG I don't see anywhere this funtion is called with $strRemoteType=NONE. Is NONE used anymore? If so, what is it's purpose?